src/yadism/output.py

```diff
@@ -100,17 +102,25 @@ def apply_pdf_theory(self, lhapdf_like, theory):
         method = runcards.Legacy.MOD_EV2METHOD.get(theory["ModEv"], theory["ModEv"])
         method = dictlike.load_enum(types.EvolutionMethod, method)
         method = couplings_mod_ev(method)
-        masses = [mq.value**2 for mq in new_eko_theory.quark_masses]
-        thresholds_ratios = np.power(list(iter(new_eko_theory.matching)), 2.0)
+        masses = [mq**2 for mq, _ in new_eko_theory.heavy.masses]
+        thresholds_ratios = np.power(new_eko_theory.heavy.matching_ratios, 2)
         sc = Couplings(
             couplings=new_eko_theory.couplings,
             order=new_eko_theory.order,
             method=method,
             masses=masses,
-            hqm_scheme=new_eko_theory.quark_masses_scheme,
-            thresholds_ratios=thresholds_ratios,
+            hqm_scheme=new_eko_theory.heavy.masses_scheme,
+            thresholds_ratios=thresholds_ratios.tolist(),
+        )
+        atlas = Atlas(
+            matching_scales=MatchingScales(masses * thresholds_ratios),
+            origin=(theory["Q0"] ** 2, theory["nf0"]),
```

Comment:
```suggestion
            origin=(theory["Qref"] ** 2, theory["nfref"]),
```
this is for $\alpha_s$